google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -137,16 +178,18 @@ void DeleteInstance(google::cloud::bigtable::InstanceAdmin instance_admin,
 //! [delete instance]
 
 //! [create cluster]
-// Before creating cluster, need to create instance first, then create cluster
-// on it.
+// Before creating cluster, need to create a production instance first,
+// then create cluster on it.
 void CreateCluster(google::cloud::bigtable::InstanceAdmin instance_admin,
                    int argc, char* argv[]) {
-  if (argc != 3) {
-    throw Usage{"create-cluster: <project-id> <instance-id> <cluster-id>"};
+  if (argc != 4) {
+    throw Usage{
+        "create-cluster: <project-id> <instance-id> <cluster-id> <zone>"};
   }
 
   google::cloud::bigtable::InstanceId instance_id(ConsumeArg(argc, argv));
   google::cloud::bigtable::ClusterId cluster_id(ConsumeArg(argc, argv));
+  std::string const zone = ConsumeArg(argc, argv);
```

Comment:
Should we remove `location` and use `zone` below?

Reply:
removed the `location` and used `zone`.